docs/site-components-deployment.md

```diff
@@ -80,10 +80,18 @@ CLIENT_URL=https://tf2pickup.fi
 BOT_NAME=tf2pickup.fi
 
 # MongoDB
-MONGODB_URI=mongodb://tf2pickup:yoursuperfunnypassword@mongodb:27017/tf2pickup
+# The commented values below were used in the old version of the tf2pickup.org project, deprecated since server 8.x version
```

Comment:
You should probably not to not use these unless on a version less than version 8.